app/model/Payment/MailingService.php

```diff
@@ -131,20 +128,13 @@ public function sendTestMail(int $groupId) : string
     /**
      * @throws BankAccountNotFound
      * @throws InvalidBankAccount
-     * @throws InvalidEmail
+     * @throws PaymentHasNoEmails
      * @throws InvalidOAuth
      * @throws UserNotFound
      * @throws OAuthNotSet
      */
     private function sendForPayment(Payment $paymentRow, Group $group, EmailTemplate $template) : void
     {
-        array_map(function (EmailRecipient $emailRecipient) : void {
-            $email = $emailRecipient->getEmailAddress();
-            if (! Validators::isEmail($email)) {
```

Comment:
Proč tu kontrolu odebíráš? ja bych se na ni naopak chtěl spolehnout, že když pouziji VO, tak je to zvalidované

Reply:
@sinacek Odebral jsem ji právě proto, že v tuhle chvíli víme, že jsou to validní emaily, protože EmailAddress nemůže obsahovat nevalidní email. V tom je ta síla VO

Reply:
ok